soccer/planning/trajectory/Trajectory.cpp

```diff
@@ -0,0 +1,43 @@
+#include "Trajectory.hpp"
+
+namespace Trajectory {
+
+int Trajectory::getNumEvals(double duration, double granularity) {
+    auto iterations = std::ceil(duration / granularity) + 1;
+    return static_cast<int>(iterations);
+}
+
+std::unique_ptr<InterpolatedPath> Trajectory::rasterizePath(
```

Comment:
This is probably fine for now, but this adds an extra copy of the trajectory that isn't necessarily needed.